plugins/services/src/js/service-configuration/PodPlacementConstraintsConfigSection.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import ConfigurationMapSection from '../../../../../src/js/components/ConfigurationMapSection';
+import ConfigurationMapTable from '../components/ConfigurationMapTable.js';
+import ConfigurationMapHeading from '../../../../../src/js/components/ConfigurationMapHeading';
+import ConfigurationVisible from '../components/ConfigurationVisible';
+
+module.exports = ({appConfig}) => {
+  let constraints = appConfig.scheduling && appConfig.scheduling.placement
```

Comment:
you can use: `let constraints = findNestedPropertyInObject(appConfig, 'scheduling.placement.constraints')` for this